### ISSUE TITLE
Do you clear the old entries from the chain ?

### ISSUE THREAD

**Issue by bobymicroby** [tam7t/droplan#35]
Hi @tam7t 

I had a quick look ad the code, and it seems to me that you only append rules to the chain.
Droplets are ephemeral, so we will end up with a lot of allowed droplets, even after we have destroyed them and they already belong to Eve.

Regards, B.

**Comment by tam7t** [comment]
Each run of `droplan` will first clear the chain before adding peers:

https://github.com/tam7t/droplan/blob/master/tables.go#L43

**Comment by bobymicroby** [comment]
Great . Go is still a bit cryptic for me :)

**Comment by josegonzalez** [comment]
Does this mean that there is a period of time during which the network is left unsecured? Seems like a better tact would be to load the existing chain, modify it in-memory, and apply changes as necessary.

**Comment by tam7t** [comment]
No, the chain that is cleared only has the `ACCEPT` rules, the default rule is `DROP` on the interface.

**Comment by josegonzalez** [comment]
So then network traffic will blip?

**Comment by tam7t** [comment]
@josegonzalez that is addressed by https://github.com/tam7t/droplan/issues/11 with the
```-A INPUT -i eth1 -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT```
rule so that established connections are not dropped.

**Comment by josegonzalez** [comment]
Right but I guess new connections will break during the time it takes to update the chain, correct?
